meson.build

```diff
@@ -11,34 +11,52 @@ version_date='2020-06-22'
 ratbagd_api_version = 1
 
 
+# Options
+enable_runtime_depcheck = get_option('runtime_depcheck')
+enable_tests = get_option('tests')
+
 # Dependencies
-dependency('pygobject-3.0', required: true)
-ratbagd = find_program('ratbagd', required: false)
-ratbagd_required_version='0.14'
-
-if not ratbagd.found()
-    message('''
-    *********************** WARNING **********************
-    * _???_     Dependency libratbag not found in $PATH! *
-    *(_)_(_)    ratbagd must be installed and running    *
-    * (o o)     for Piper to work.                       *
-    *==\o/==                                             *
-    ******************************************************
-    ''')
-else
-    r = run_command(ratbagd, '--version')
-    if r.returncode() != 0 or r.stdout().version_compare('<' + ratbagd_required_version)
-    message('''
-    *********************** WARNING ****************************
-    *                 (   )(                                   *
-    *         (\-.     ) (  )    ratbagd found in $PATH is too *
-    *         / _`> .---------.  old and  will  not  work with *
-    * _)     / _)=  |'-------'|  this version of Piper. Please *
-    *(      / _/    |O   O   o|  update libratbag/ratbagd.     *
-    * `-.__(___)_   | o O . o |                                *
-    ************************************************************
-    ''')
+if enable_runtime_depcheck
+    ratbagd = find_program('ratbagd', required: false)
+    ratbagd_required_version='0.14'
+
+    if not ratbagd.found()
+        message('''
+        *********************** WARNING **********************
+        * _???_     Dependency libratbag not found in $PATH! *
+        *(_)_(_)    ratbagd must be installed and running    *
+        * (o o)     for Piper to work.                       *
+        *==\o/==                                             *
+        ******************************************************
+        ''')
+    else
+        r = run_command(ratbagd, '--version')
+        if r.returncode() != 0 or r.stdout().version_compare('<' + ratbagd_required_version)
+        message('''
+        *********************** WARNING ****************************
+        *                 (   )(                                   *
+        *         (\-.     ) (  )    ratbagd found in $PATH is too *
+        *         / _`> .---------.  old and  will  not  work with *
+        * _)     / _)=  |'-------'|  this version of Piper. Please *
+        *(      / _/    |O   O   o|  update libratbag/ratbagd.     *
+        * `-.__(___)_   | o O . o |                                *
+        ************************************************************
+        ''')
+        endif
```

Comment:
in two minds about this. I'm wondering whether to make this required, after all we have the  dependency-check option now.

Reply:
Sounds reasonable, but how should we handle it if the version is smaller than `ratbagd_required_version`? Aborting or just a warning? When thinking about bug triaging, aborting seems like the better idea.

Reply:
abort. If the user builds piper but has an old ratbagd version running, you're guaranteed to blow up later anyway, might as well make this look more intended.

The chance of the user restarting into a different ratbagd before starting this compiled piper version is so minuscule as to be statistical noise.